* Add 1000 to LINUX_API version. Avoids crashes between branch devel and cofb.
* Adjustments between current branch devel and the implemetation of cofb
  to clean apply the follow patch. These changes needs to fix later.

Index: linux-2.6.26.8-source/include/linux/cooperative.h
===================================================================
--- linux-2.6.26.8-source.orig/include/linux/cooperative.h
+++ linux-2.6.26.8-source/include/linux/cooperative.h
@@ -20,7 +20,7 @@
 
 #include <asm/cooperative.h>
 
-#define CO_LINUX_API_VERSION    14
+#define CO_LINUX_API_VERSION    1013
 
 #pragma pack(0)
 
@@ -235,7 +235,7 @@
 	CO_OPERATION_CONSOLE_SET_ORIGIN,
 	CO_OPERATION_CONSOLE_SAVE_SCREEN,
 	CO_OPERATION_CONSOLE_INVERT_REGION,
-	CO_OPERATION_CONSOLE_CONFIG,
+	CO_OPERATION_CONSOLE_SIZES,
 } co_operation_console_t;
 
 
@@ -256,7 +256,6 @@
 			co_console_unit top;
 			co_console_unit bottom;
 			co_console_unit lines;
-			co_console_character charattr;
 		} scroll;
 		struct {
 			co_console_unit y;
@@ -292,8 +291,8 @@
 		struct {
 			co_console_unit rows;
 			co_console_unit cols;
-			co_console_unit attr;
-		} config;
+			co_console_unit backbuf;
+		} sizes;
 		co_cursor_pos_t cursor;
 	};
 } __attribute__((packed)) co_console_message_t;
Index: linux-2.6.26.8-source/drivers/video/console/cocon.c
===================================================================
--- linux-2.6.26.8-source.orig/drivers/video/console/cocon.c
+++ linux-2.6.26.8-source/drivers/video/console/cocon.c
@@ -467,7 +467,7 @@
 	co_send_message_restore(flags);
 }
 
-static int cocon_resize(struct vc_data *vc, unsigned int width,
+static int cocon_resize(struct vc_data *vc, unsigned int width, 
 			unsigned int height, unsigned int user)
 {
 	return -EINVAL;
